src/net.h

```diff
@@ -674,8 +675,16 @@ class V1TransportDeserializer final : public TransportDeserializer
     }
 
 public:
+<<<<<<< HEAD
+    V1TransportDeserializer(const CChainParams& chain_params_in, int nTypeIn, int nVersionIn) : m_chain_params(chain_params_in), hdrbuf(nTypeIn, nVersionIn), vRecv(nTypeIn, nVersionIn)
+    {
+||||||| parent of f070a97fe5... Remove header checks out of net_processing
```

Comment:
I think something has gone wrong here

Reply:
oof, thanks

Reply:
@jonatack the correct fix here is below but I'll hold off on touching anything until you're ready.

```suggestion
    V1TransportDeserializer(const CChainParams& chain_params_in, const NodeId node_id_in, int nTypeIn, int nVersionIn) : m_chain_params(chain_params_in), node_id(node_id_in), hdrbuf(nTypeIn, nVersionIn), vRecv(nTypeIn, nVersionIn)
    {
```

Reply:
thanks @troygiorshev